package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_bizarre_cravings",
```

Comment:
Consider adding a more descriptive project name instead of "s47_bizarre_cravings". This will make it easier for others to understand the purpose of the project. [important]